SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -128,6 +128,10 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	UFUNCTION(BlueprintPure, Category = "Spatial Functional Test")
 	int GetNumberOfClientWorkers();
 
+	// Convenience function that returns the Id used for executing steps on all Servers / Clients
+	UFUNCTION(BlueprintPure, meta = (ToolTip = "Returns the Id (0) that represents all Workers (ie Server / Client), useful for when you want to have a Server / Client Step run on all of them"), Category = "Spatial Functional Test")
+	int GetAllWorkersId() { return FWorkerDefinition::ALL_WORKERS_ID; }
```

Comment:
I am not that knowledgeable about BPs, but can't you have an enum accessible from BPs? Not sure if that will allow arbitrary IDs, though.

Reply:
You can have enums in BP, but why would you want an enum? At the moment those functions receive an int given that the number of workers vary. If we'd do something with enum then it would have to be converted to int, so it would be 2 nodes in the BP.

Reply:
Yeah, thought it could be something annoying like that.
Just that having enums would be helpful if we ever introduced more special values for these in the future (as they would be grouped together).

Reply:
If that ever comes in the future, we can always add duplicate functions that would receive an enum. ATM don't think it's worth it.